scanners/ffuf/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+name: ffuf
+description: A Helm chart for the ffuf security Scanner that integrates with the secureCodeBox.
+
+type: application
+# version - gets automatically set to the secureCodeBox release version when the helm charts gets published
+version: v3.1.0-alpha1
+appVersion: "v1.5.0"
+kubeVersion: ">=v1.11.0-0"
+
```

Comment:
The following is missing:
```yaml
annotations:
  versionApi: https://api.github.com/repos/ffuf/ffuf/releases/latest
  # supported cpu architectures for which docker images for the scanner should be build
  supported-platforms: linux/amd64
```
SCB Bot requires a github version api to figure out if the scanner needs upgrading. This is set in `versionApi` Also adding ffuf to list of scanners to check for upgrades is needed and is done [here](https://github.com/secureCodeBox/secureCodeBox/blob/f25f9c6283d5e820e44ca07508adf76102896558/.github/workflows/scb-bot.yaml#L13).
`supported-platforms` is self-explanatory.

Reply:
Thanks for the hint 👌 I probably took it from screenshoter/Chart.yaml. Should it be fixed there, too?

Reply:
- No, screenshooter is one of our own developed scanners. So checking if a newer version exists is not necessary. The version of screenshooter follows the version of SCB.
- Adding ffuf to list of scanners to check for upgrades is needed and is done [here](https://github.com/secureCodeBox/secureCodeBox/blob/f25f9c6283d5e820e44ca07508adf76102896558/.github/workflows/scb-bot.yaml#L13).